### PR TITLE
Cluster

### PR DESCRIPTION
**Pull request by TJKoury**
Multi process modification, updated for 0.15.1.

### CODE REVIEW

File: red/api/flows.js [hunk 1]
```diff
@@ -79,5 +80,9 @@ module.exports = {
                 res.status(500).json({error:"unexpected_error", message:err.message});
             });
         }
+       
+        if(cluster.isWorker){
+
```
Comment (dceejay) [inline red/api/flows.js]: needed ? and if not ... line 20 also ?

Comment (TJKoury) [inline red/api/flows.js]: YES, it's the key to the entire implementation!

...actually no, that was just a left over from a merge.  Good catch!

**Comment by TJKoury** [comment]
@dceejay @knolleary I've got the tests working, just polishing up a few nodes.

**Comment by vielmetti** [comment]
@TJKoury I am interested in this work, any suggestions on how and where to follow progress beyond this set of patches would be welcomed.

**Comment by dceejay** [comment]
Follow along in the cluster thread on Slack

**Comment by TJKoury** [comment]
Closing this pull request and opting for a new approach that clusters the admin interface (along with the admin server).  Based on conversations with @knolleary and @dceejay about expected default behaviors and backwards compatibility.

**Pull request closed without merging**